blazor/heatmap-chart/selection.md

```diff
@@ -48,4 +56,90 @@ In the [Blazor HeatMap Chart](https://www.syncfusion.com/blazor-components/blazo
 
 ```
 
-![Selection in Blazor HeatMap Chart](images/blazor-heatmap-chart-selection.png)
\ No newline at end of file
+![Selection in Blazor HeatMap Chart](images/blazor-heatmap-chart-selection.png)
+
+The image below illustrates the process of selecting multiple cells in the HeatMap by clicking and dragging the mouse across the specific cells.
+
+![Multiple Selection in Blazor HeatMap Chart](images/blazor-heatmap-chart-multiple-selection.gif)
+
+## Enable single cell selection
+
+In the HeatMap, the [EnableMultiSelect]() property is used to allow single cell selection. When you set the `EnableMultiSelect` property to **false**, only one cell is selected. By default, `EnableMultiSelect` property is set to **true**.
+
+```cshtml
+
+@using Syncfusion.Blazor.HeatMap
+
+<SfHeatMap DataSource="@DataSource"AllowSelection="true" EnableMultiSelect="false">
+    <HeatMapTitleSettings Text="Sales Revenue per Employee (in 1000 US$)"></HeatMapTitleSettings>
+    <HeatMapXAxis Labels="@XLabels"></HeatMapXAxis>
+    <HeatMapYAxis Labels="@YLabels"></HeatMapYAxis>
+</SfHeatMap>
+
+@code{
+    public int YIndex = 0;
+    public string[] XLabels = new string[] { "Nancy", "Andrew", "Janet", "Margaret", "Steven", "Michael", "Robert", "Laura", "Anne", "Paul", "Karin", "Mario" };
+    public string[] YLabels = new string[] { "Mon", "Tues", "Wed", "Thurs", "Fri", "Sat" };
+    public double[,] DataSource = new double[,]
+    {
+         { 73, 39, 26, 39, 94, 0 },
+         { 93, 58, 53, 38, 26, 68 },
+         { 99, 28, 22, 4, 66, 90 },
+         { 14, 26, 97, 69, 69, 3 },
+         { 7, 46, 47, 47, 88, 6 },
+         { 41, 55, 73, 23, 3, 79 },
+         { 56, 69, 21, 86, 3, 33 },
+         { 45, 7, 53, 81, 95, 79 },
+         { 60, 77, 74, 68, 88, 51 },
+         { 25, 25, 10, 12, 78, 14 },
+         { 25, 56, 55, 58, 12, 82 },
+         {74, 33, 88, 23, 86, 59 }
+    };
+}
+
+```
+![Single cell selection in Blazor HeatMap Chart](images/blazor-heatmap-chart-single-cell-selection.gif)
+
+### ClearSelection Method
+
+The `ClearSelection` method is used to deselect the selected cells in the HeatMap. To clear the selection, reset the selected cells data structure to an empty state. The below example illustrates the same.
```

Comment:
The `ClearSelection` method can be used to clear the selected cells. The below example illustrates the same.

Use the API link.